saleor/graphql/product/tests/test_product.py

```diff
@@ -5546,6 +5546,42 @@ def test_update_product_with_page_reference_attribute_value(
     assert product_type_page_reference_attribute.values.count() == values_count + 1
 
 
+def test_update_product_with_empty_input_collections(
+    product, permission_manage_products, staff_api_client
+):
+    # given
+    query = """
+    mutation updateProduct($productId: ID!, $input: ProductInput!) {
+      productUpdate(id: $productId, input: $input) {
+        productErrors {
+          field
+          message
+          code
+        }
+        product {
+          id
+        }
+      }
+    }
+
+    """
+    product_id = graphene.Node.to_global_id("Product", product.pk)
+    variables = {
+        "productId": product_id,
+        "input": {"collections": [""]},
+    }
+    # when
+    response = staff_api_client.post_graphql(
+        query, variables, permissions=[permission_manage_products]
+    )
+    # then
+    content = get_graphql_content(response)
+    data = content["data"]["productUpdate"]
+    assert len(data["productErrors"]) == 1
+    productErrors = data["productErrors"][0]
+    assert productErrors["code"] == "GRAPHQL_ERROR"
```

Comment:
In tests, we are using snake case so for consistency it should be:
```suggestion
    product_errors = data["productErrors"][0]
    assert product_errors["code"] == "GRAPHQL_ERROR"
```

Also, we usually compare error code to value from the error code enum, so in this case it should be:
```suggestion
    assert product_errors["code"] == ProductErrorCode.GRAPHQL_ERROR.name
```